tutorials/Tutorial6_Better_Retrieval_via_DPR.ipynb

```diff
@@ -384,11 +384,11 @@
    "source": [
     "# !pip install git+https://github.com/deepset-ai/haystack.git#egg=farm-haystack[milvus]\n",
     "\n",
-    "#from haystack.utils import launch_milvus\n",
-    "#from haystack.document_stores import MilvusDocumentStore\n",
+    "# from haystack.utils import launch_milvus\n",
```

Comment:
The tutorial 6 changes from https://github.com/deepset-ai/haystack/pull/2148 made it in here. We shouldn't do that. Could you please remove them?